packages/ampd-sdk/src/grpc/client.rs

```diff
@@ -317,11 +292,15 @@ mod tests {
         }
     }
 
-    async fn setup_test_client(
+    async fn test_setup(
```

Comment:
I know we use this naming scheme throughout many parts of the codebase, but I'm not a big fan of it, because it tells you nothing about what's happening. Furthermore, it seems like at least most tests don't need the connection and abort handle, so the abstraction doesn't fit the need. Better to have a separate setup for the few tests that do need them